builtin/providers/openstack/provider.go

```diff
@@ -105,6 +105,11 @@ func Provider() terraform.ResourceProvider {
 			"openstack_lb_monitor_v1":                  resourceLBMonitorV1(),
 			"openstack_lb_pool_v1":                     resourceLBPoolV1(),
 			"openstack_lb_vip_v1":                      resourceLBVipV1(),
+			"openstack_lbaas_loadbalancer_v2":          resourceLoadBalancerV2(),
```

Comment:
I get that LBaaS v1 used the `/lb/` URL and LBaaS v2 uses the `lbaas` URL, so I understand why you chose to use `lbaas` in the name... but I wonder if the resources should keep `lb` and `LB` in their names for consistency... Thoughts?